src/client-agent/request.c

```diff
@@ -259,7 +259,7 @@ void * req_receiver(__attribute__((unused)) void * arg) {
         } else if (strncmp(node->target, "logcollector", 12) == 0) {
             length = lccom_dispatch(node->buffer, &buffer);
         } else if (strncmp(node->target, "com", 3) == 0) {
```

Comment:
I think I'm missing something, but where is the `upgrade` key that replaces `com`?